components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -1572,6 +1580,14 @@ paths:
         200:
           description: |
             OK.
+        201:
```

Comment:
I believe, the resource returns 201 response when successfully restoring the revision. If so, shall we remove the 200 response from API definition?

Reply:
Also, REST API impl also need to return the 201 using `Response.created()` instead of `Response.ok()`

Reply:
ack, yes, will remove

Reply:
resolved